static/src/js/containers/CollectionResultsContainer/CollectionResultsContainer.scss

```diff
@@ -12,16 +12,25 @@
   position: absolute;
   bottom: 100%;
   left: 0;
-  padding: $spacer 1.25rem;
   background: $color__white;
 }
 
+.coll-res__tab-handle {
+  position: absolute;
+  top: 0;
+  height: 10px;
+  width: 100%;
+  background: $color__white url('../../../assets/images/drag.png') no-repeat center 5px;
```

Comment:
Is there an option to use a variable or something to define the asset path instead of having to back out 3 folders?

Reply:
There maybe something fancy in webpack, but a var wouldnt be super useful as it would need to be different for every level deep. I will look into any improvements we can make on the webpack side. I agree, its pretty ugly.